src/databricks/labs/remorph/resources/reconcile/dashboards/reconciliation_metrics/07_0_schema_details_table.sql

```diff
@@ -0,0 +1,36 @@
+/* --title 'Schema Details' --width 6 */
+WITH tmp AS (
+    SELECT
+        recon_table_id,
+        inserted_ts,
+        explode(data) AS schema_data
+    FROM
+        remorph.reconcile.details
+    WHERE
+        recon_type = 'schema'
+)
+SELECT
+    main.recon_id,
+    main.source_table.`catalog` AS source_catalog,
+    main.source_table.`schema` AS source_schema,
+    main.source_table.table_name AS source_table_name,
+    CASE
+        WHEN COALESCE(main.source_table.catalog, '') <> '' THEN CONCAT(main.source_table.catalog, '.', main.source_table.schema, '.', main.source_table.table_name)
```

Comment:
This case statement can be converted as below, avoiding the Unnecessary use of `Coalesece`
```
 IF(
                     ISNULL(source_catalog),
                     CONCAT_WS('.', source_schema, source_table_name),
                     CONCAT_WS(
                             '.',
                             source_catalog,
                             source_schema,
                             source_table_name
                     )
             ) AS source_table,
```